core/src/main/java/org/apache/druid/java/util/metrics/JvmMonitor.java

```diff
@@ -146,7 +151,25 @@ private void emitDirectMemMetrics(ServiceEmitter emitter)
 
   private void emitGcMetrics(ServiceEmitter emitter)
   {
-    gcCounters.emit(emitter, dimensions);
+    if (gcCounters != null) {
+      gcCounters.emit(emitter, dimensions);
+    }
+  }
+
+  @Nullable
+  private GcCounters getGcCounters()
+  {
+    try {
+      return new GcCounters();
+    }
+    catch (RuntimeException e) {
+      // in JDK11 jdk.internal.perf.Perf is not accessible, unless
+      // --add-exports java.base/jdk.internal.perf=ALL-UNNAMED is set
+      log.warn(e, "Cannot initialize GC counters. If running JDK11 and above,"
```

Comment:
Better to remove the `e`, it makes the logs needlessly noisy. (It doesn't seem likely that the stack trace will provide useful info beyond what the log message provides.)

Reply:
done